dev/com.ibm.ws.microprofile.rest.client_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -0,0 +1,256 @@
+            <artifactId>jaxrs20.client</artifactId>
+            <version>1.0</version>
+            <scope>system</scope>
+            <systemPath>${com.ibm.ws.jaxrs.2.0.client_}</systemPath>
+        </dependency> -->
+
+        <dependency>
+            <groupId>com.ibm.ws</groupId>
+            <artifactId>jaxrs20.cdi</artifactId>
+            <version>1.0</version>
+            <scope>system</scope>
+            <systemPath>${com.ibm.ws.jaxrs.2.0.cdi_}</systemPath>
+        </dependency>
+
+        <dependency>
+            <groupId>com.ibm.ws</groupId>
+            <artifactId>logging</artifactId>
+            <version>1.0</version>
+            <scope>system</scope>
+            <systemPath>${com.ibm.ws.logging_}</systemPath>
+        </dependency>
+
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>1.2</version>
+            <scope>system</scope>
+            <systemPath>${com.ibm.websphere.javaee.annotation.1.2_}</systemPath>
+        </dependency>
+
+        <dependency>
+            <groupId>javax.ws.rs</groupId>
+            <artifactId>javax.ws.rs-api</artifactId>
+            <version>2.0</version>
+            <!--  <scope>provided</scope> -->
+            <scope>system</scope>
+            <systemPath>${com.ibm.websphere.javaee.jaxrs.2.0_}</systemPath>
+            <!-- <systemPath>/home/andymc/mpRestClientTCK/open-liberty/dev/build.image/wlp/dev/api/spec/com.ibm.websphere.javaee.jaxrs.2.0_1.0.19.jar</systemPath> -->
+        </dependency>
+
+        <dependency>
+            <groupId>javax.enterprise</groupId>
+            <artifactId>cdi-api</artifactId>
+            <version>1.2</version>
+            <!-- <scope>provided</scope> -->
+            <scope>system</scope>
+            <systemPath>${com.ibm.websphere.javaee.cdi.1.2_}</systemPath>
+        </dependency>
+
+        <dependency>
+            <groupId>javax.json</groupId>
+            <artifactId>javax.json-api</artifactId>
+            <version>1.0</version>
+            <!-- <scope>provided</scope> -->
+            <scope>system</scope>
+            <systemPath>${com.ibm.websphere.javaee.jsonp.1.0_}</systemPath>
+        </dependency>
+
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>6.9.9</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.jboss.arquillian.testng</groupId>
+            <artifactId>arquillian-testng-container</artifactId>
+            <version>${arquillian.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.ibm.ws.org.jboss.arquillian.container</groupId>
+            <artifactId>arquillian-wlp-managed-8.5</artifactId>
+            <!-- <version>1.0.0.Final-SNAPSHOT</version> -->
+            <version>1.0.0.Final-20180108.155248-1</version>
```

Comment:
I think this might break if the 'old' snapshot is removed (which is quite likely).